typhos/display.py

```diff
@@ -781,6 +782,7 @@ def display_widget(self):
     @staticmethod
     def _get_templates_from_macros(macros):
         ret = {}
+        paths = [cache.get_global_path_cache()]
```

Comment:
Something seems strange here. This is a list of one item, the path cache, and it is being passed into `utils.find_file_in_paths` in place of what is expected to be an iterable of paths.

Reply:
I wanted an escape-hatch from the path deduplication in `find_file_in_paths`, which is how this implementation resulted. 

Alternative in: c993a1b